sapi/fpm/fpm/fpm_log.c

```diff
@@ -136,6 +149,10 @@ int fpm_log_write(char *log_format) /* {{{ */
 		}
 		proc = *proc_p;
 		fpm_scoreboard_proc_release(proc_p);
+
+		if (UNEXPECTED(fpm_access_log_suppress(&proc) == 1)) {
```

Comment:
NIT: guess it might be more optimal to just check it like
```
if (UNEXPECTED(fpm_access_log_suppress(&proc)))
```
and for all failures return 0 instead of those negative values that are not being used anywhere...